manifests/m/Microsoft/SQLServerManagementStudio/15.0.18386.0/Microsoft.SQLServerManagementStudio.installer.yaml

```
@@ -0,0 +1,32 @@
+# Automatically updated by the winget bot at 2021/Jul/28
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.0.0.schema.json
+PackageIdentifier: Microsoft.SQLServerManagementStudio
+PackageVersion: 15.0.18386.0
+MinimumOSVersion: 10.0.0.0
+Platform:
+  - Windows.Desktop
+FileExtensions:
+  - SQL
+InstallModes:
+  - interactive
+  - silent
+  - silentWithProgress
+InstallerSwitches:
+  Silent: /install /quiet /norestart
+  SilentWithProgress: /install /passive /norestart
+  Interactive: /install
+Installers:
+  - Architecture: x64
+    InstallerLocale: en-US
+    Scope: machine
+    InstallerType: exe
+    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-ENU.exe
+    InstallerSha256: 03E567B7475EC2E2DA237D70E9A833DEABF52CA66D521F340A1EBC8FDEA91170
+  - Architecture: x64
+    InstallerLocale: zh-CN
+    Scope: machine
+    InstallerType: exe
+    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-CHS.exe
+    InstallerSha256: 7266B8E85E2AA843829315DF85D176E5E618B6FA7D2534481D9BB7F2CF0880FB
```

Corrects the InstallerType, adds /norestart to the Custom field as a temporary fix until they can detect reboot behavior 
```suggestion
InstallerSwitches:
  Custom: /norestart
Installers:
  - Architecture: x64
    InstallerLocale: en-US
    Scope: machine
    InstallerType: burn
    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-ENU.exe
    InstallerSha256: 03E567B7475EC2E2DA237D70E9A833DEABF52CA66D521F340A1EBC8FDEA91170
  - Architecture: x64
    InstallerLocale: zh-CN
    Scope: machine
    InstallerType: burn
    InstallerUrl: https://download.microsoft.com/download/1/e/c/1ec92162-142d-4fed-a575-6e2195b65a66/SSMS-Setup-CHS.exe
    InstallerSha256: 7266B8E85E2AA843829315DF85D176E5E618B6FA7D2534481D9BB7F2CF0880FB
```